nilimamaka/SpringBoot/MultipleDBConnection/src/main/java/com/example/multipledbconnection/fileDetails/configure/ConfigureFD.java

```diff
@@ -0,0 +1,78 @@
+package com.example.multipledbconnection.fileDetails.configure;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.Configuration;
+import org.springframework.context.annotation.Primary;
+import org.springframework.core.env.Environment;
+import org.springframework.data.jpa.repository.config.EnableJpaRepositories;
+import org.springframework.jdbc.datasource.DriverManagerDataSource;
+import org.springframework.orm.jpa.JpaTransactionManager;
+import org.springframework.orm.jpa.JpaVendorAdapter;
+import org.springframework.orm.jpa.LocalContainerEntityManagerFactoryBean;
+import org.springframework.orm.jpa.vendor.HibernateJpaVendorAdapter;
+import org.springframework.transaction.PlatformTransactionManager;
+import org.springframework.transaction.annotation.EnableTransactionManagement;
+
+import javax.sql.DataSource;
+import java.util.HashMap;
+import java.util.Map;
+
+@Configuration
+@EnableTransactionManagement
+@EnableJpaRepositories(
+        entityManagerFactoryRef = "entityManagerFactoryBeanFD",
+        transactionManagerRef = "transactionManagerFD",
+        basePackages ="com.example.multipledbconnection.fileDetails.repo"
+)
+public class ConfigureFD {
```

Comment:
Will be better , if you set filename more contextual.
Eg. FileDatabaseConfig or FileDbConfig instead ConfigureFD